src/plugins/auto/multi_schedule.cpp

```diff
@@ -244,16 +244,7 @@ MultiSchedule::~MultiSchedule() {
             IE_ASSERT(count == reqAllEndTimes.size());
             reqAllStartTimes.sort(std::less<Time>());
             reqAllEndTimes.sort(std::less<Time>());
-            if (_workerRequest.first == "CPU_HELP") {
-                LOG_INFO_TAG("CPU_HELP:infer:%ld", _cpuHelpInferCount + count);
-                if (_cpuHelpFps > 0.0) {
-                    LOG_INFO_TAG("CPU_HELP:fps:%lf", _cpuHelpFps);
-                } else if (count >= 1) {
-                    std::chrono::duration<double, std::milli> durtation =
-                        reqAllEndTimes.back() - reqAllStartTimes.front();
-                    LOG_INFO_TAG("CPU_HELP:fps:%lf", count * 1000 / durtation.count());
-                }
-            } else {
+            if (_workerRequest.first != "CPU_HELP") {
```

Comment:
Could we update fps calculation for CPU_HELP based completed frames instead of totally drop it?

Reply:
we re not dop it , the calc is in above code , current master cannot give correct cpu help fps